http/document_service.go

```diff
@@ -194,7 +194,9 @@ func (h *DocumentHandler) handleGetDocuments(w http.ResponseWriter, r *http.Requ
 
 	opt := influxdb.AuthorizedWhere(a)
 
```

Comment:
imo would be nice to have a check here to see if both `Org` and `OrgID` and return an error, to avoid ambiguous behavior in that situation.

Reply:
👍